site/content/components/collaborators.jsx

```diff
@@ -0,0 +1,50 @@
+export default function Collaborators() {
+  return (
```

Comment:
I feel we probably want to implement this stuff as loop over an array of collaborators e.g.

const collaborators = [
  {
	title:
	url: 
        image: 
  }
]

...

<section>
  ....
  for collaborator in collaborators ...
```

collaborators could even be loaded from a data file ... (but that's the future)

Reply:
This is a list now.